packages/nvidia-container-toolkit/nvidia-container-toolkit.spec

```
@@ -0,0 +1,56 @@
+%global debug_package %{nil}
+
+%global goproject github.com/NVIDIA
+%global gorepo nvidia-container-toolkit
+%global goimport %{goproject}/%{gorepo}
+
+%global gover 1.7.0
+%global rpmver %{gover}
+
+Name: %{_cross_os}nvidia-container-toolkit
+Version: %{rpmver}
+Release: 1%{?dist}
+Summary: Tool to build and run GPU accelerated containers
+License: Apache-2.0
+URL: https://%{goimport}
+
+Source0: https://%{goimport}/archive/v%{gover}/v%{gover}.tar.gz
+Source1: nvidia-container-toolkit-config.toml
+Source2: nvidia-container-toolkit-tmpfiles.conf
+Source3: nvidia-oci-hooks-json
+
+BuildRequires: %{_cross_os}glibc-devel
+Requires: %{_cross_os}libnvidia-container
+Requires: %{_cross_os}shimpei
+
+%description
+%{summary}.
+
+%prep
+%autosetup -n %{gorepo}-%{gover} -p1
+%cross_go_setup %{gorepo}-%{gover} %{goproject} %{goimport}
+
+%build
+%cross_go_configure %{goimport}
+go build -buildmode=pie -ldflags=-linkmode=external -o nvidia-container-toolkit ./cmd/nvidia-container-toolkit
```

This should fix the rpm build failure that turning off the debug package fixed:

```suggestion
go build -buildmode=pie -ldflags="${GOLDFLAGS}" -o nvidia-container-toolkit ./cmd/nvidia-container-toolkit
```